jobs/check-aws-regions.Jenkinsfile

```
@@ -0,0 +1,31 @@
+import org.yaml.snakeyaml.Yaml;
+
+node {
+    checkout scm
+    // these are script global vars
+    pipeutils = load("utils.groovy")
+}
+
+properties([
+    pipelineTriggers([
+        githubPush(),
+        // once a day
+        pollSCM('H H * * *')
+    ]),
+    buildDiscarder(logRotator(
+        numToKeepStr: '100',
+        artifactNumToKeepStr: '100'
+    )),
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+])
+
+cosaPod(){
+    withCredentials([file(variable: 'AWS_CONFIG_FILE',
+                           credentialsId: 'aws-build-upload-config')]) {
+        def disabled_regions = shwrapCapture("ore aws list-regions --disabled")
+        if (disabled_regions != "None") {
+            echo "Disabled regions: ${disabled_regions}"
+            return
+        }    
```

we do still want to send a slack message if there are disabled regions
(so we know to do something). 
```suggestion
    def disabled_regions = ""
    withCredentials([file(variable: 'AWS_CONFIG_FILE',
                           credentialsId: 'aws-build-upload-config')]) {
        disabled_regions = shwrapCapture("ore aws list-regions --disabled")
    }
    if (disabled_regions != "None") {
        echo "Disabled regions: ${disabled_regions}"
        trySlackSend(...
        return
    }    
```

---

yea, I have been trying to solve this error when I add the slack message. I have been hitting this error on adding the trySlackSend(). - https://jenkins-fedora-coreos-pipeline.apps.ocp.stg.fedoraproject.org/job/check-aws-regions/5/console

```
11:09:03  + oc get secret -o json slack-api-token
11:09:03  + jq -r '.metadata.annotations["jenkins.io/emoji-prefix"]'
11:09:03  Error from server (Forbidden): secrets "slack-api-token" is forbidden: User "system:serviceaccount:fedora-coreos-pipeline:default" cannot get resource "secrets" in API group "" in the namespace "fedora-coreos-pipeline"
```

---

Looks like the `serviceAccount: “jenkins”` had to be added in the cosaPod() func. That fixes the permission error.